src/api/Middleware/flightController.js

```diff
@@ -1,28 +1,82 @@
-import actionTypes from '../Actions/actionTypes';
+import { connectFlightController, disconnectFlightController, sendFlightControl } from '../Actions';
+import { actionTypes } from '../Actions/actionTypes';
+import { rootOwnerKey } from '../keys';
+
 import api from '../api';
 
-let topic;
+let topic = undefined;
```

Comment:
```suggestion
let topic;
```

This does not have to be initialized as undefined, and our linting currently warns about it.